staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -323,6 +327,12 @@ func NewKubectlCommand(o KubectlOptions) *cobra.Command {
 	if kubeConfigFlags == nil {
 		kubeConfigFlags = defaultConfigFlags
 	}
+	if os.Getenv("ENABLE_KUBERC") != "" {
```

Comment:
nitpick: could you please use https://github.com/kubernetes/kubernetes/blob/04675428bbfc9bf7ba4c9e1abfc427b6228069d9/staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go#L428